index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="container-top">
+            <h1>Rita Daniel</h1>
+            <img src="./images/Union2x.svg" alt="menu-bar">
```

Comment:
- [ ] Kindly I recommend you to use `nav` tag for  hamburger button.The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.